examples/jax/debugging/simple_spread/feedforward/decentralised/run_ippo.py

```diff
@@ -21,13 +21,16 @@
 
 import optax
 from absl import app, flags
+from jax.config import config as jax_config
 
 from mava.systems.jax import ippo
 from mava.utils.environments import debugging_utils
 from mava.utils.loggers import logger_utils
 
 # Without this flag, JAX uses the whole GPU from the beginning and our trainer crashes.
 os.environ["XLA_PYTHON_CLIENT_PREALLOCATE"] = "false"
+# Env uses int64 action space due to the use of spac.Discrete.
+jax_config.update("jax_enable_x64", True)
```

Comment:
I see that even though you updated the code Github still displays the old code. Is there an easy way to refresh this?

Reply:
Hmm not sure. I think comments probably get pinned to an old version of the code, it should say these comments are stale though since this code no longer exits. Let's investigate this in the future.